src/depot/zip.clj

```diff
@@ -5,25 +5,34 @@
 
 (defn- zip-skip-ws
   "Skip whitespace, comments, and uneval nodes."
-  ([loc]
-   (zip-skip-ws loc rzip/right rzip/end?))
-  ([loc next-fn end?-fn]
-   (loop [loc loc]
-     (if (and loc
-              (not (end?-fn loc))
-              (#{:comment :whitespace :newline :comma :uneval} (rzip/tag loc)))
-       (recur (next-fn loc))
-       loc))))
+  [loc next-fn skip-fn end?-fn]
+  (loop [loc loc]
+    (cond
+      (nil? loc) loc
+      (end?-fn loc) loc
+
+      (#{:comment :whitespace :newline :comma} (rzip/tag loc))
+      (recur (next-fn loc))
+
+      (= :uneval (rzip/tag loc))
+      (recur (skip-fn loc))
```

Comment:
Ah, makes sense, glad it's pretty well telegraphed as `uneval`. By `uneval` do you mean the `#_` reader macro? Wasn't sure what the name meant exactly.

Reply:
exactly, uneval == `#_ ...`